app/models/fae/navigation.rb

```diff
@@ -58,7 +58,7 @@ def increment_coordinates
     end
 
     def current_section
-      @current_path.gsub(/\/new|\/[0-9]\/edit/, '')
+      @current_path.gsub(/\/new|\/[0-9]*\/edit/, '')
```

Comment:
Wasn't to concerned as integers would only be used as IDs in the URL, but specificity never hurts. Updated.